tensorflow/workspace2.bzl

```diff
@@ -197,9 +197,9 @@ def _tf_repositories():
     tf_http_archive(
         name = "onednn",
         build_file = "//third_party/mkl_dnn:mkldnn_v1.BUILD",
-        sha256 = "2f76b407ef8893cca71340f88cd800019a1f14f8ac1bbdbb89a84be1370b52e3",
-        strip_prefix = "oneDNN-3.2.1",
-        urls = tf_mirror_urls("https://github.com/oneapi-src/oneDNN/archive/refs/tags/v3.2.1.tar.gz"),
+        sha256 = "be2cd865401e43254d509d6843cc5ec7bca7e032d7ddb17ebfd3184e4b14b127",
+        strip_prefix = "oneDNN-3.3-rc",
```

Comment:
When is the official release?  It seems odd to depend on a release candidate.

Reply:
The official release is expected to be end of this week. Is it okay if we update oneDNN during the next RC release?

Reply:
@cantonios I have updated the oneDNN version to v3.3. Please take a look. Thanks.

Reply:
Thanks!